bayesian/api_v1.py

```diff
@@ -642,6 +678,10 @@ def post():
                                          .format(id=request_id))
             raise HTTPError(500, "Error processing request {t}. manifest files could not be processed"
                                  .format(t=request_id))
+        try:
+            del_temp_files()
```

Comment:
If something unexpected (an exception) happens between cloning the repository and deleting the files, the files will remain there and pollute /tmp. If this happens many times, the disk space will be exhausted.

You need some mechanism, which deletes the files even in case of exception. Something like [f8a_worker.utils.tempdir](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/utils.py#L140)
Example of [usage](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/workers/git_stats.py#L25)